dev/io.openliberty.jaxb.3.0.internal.tools/resources/com/ibm/ws/jaxb/tools/internal/resources/JaxbToolsMessages.nlsprops

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+# NLS_MESSAGEFORMAT_ALL
+#
+#CMVCPATHNAME com.ibm.ws.jaxws.ejb/resources/com/ibm/ws/jaxb/tools/internal/resources/JaxbToolsMessages.nlsprops
```

Comment:
The CMVCPATHNAME should change for all of these.

Reply:
Ah, I thought # made them a comment but I'll update them.

Reply:
It does make them a comment.  But it doesn't mean you shouldn't update it to be accurate.  I see that you have done so.  Marking it resolved.